dashboard/lib/policies/user.rb

```diff
@@ -6,4 +6,23 @@ def self.user_attributes(user)
     authentication_options = user.authentication_options.map {|ao| ao.attributes.compact}
     attributes.merge('authentication_options_attributes' => authentication_options).compact
   end
+
+  def self.assign_form_params(user, params)
+    assign_auth_option_params(user, params)
+    user.assign_attributes(params.compact)
+  end
+
+  def self.assign_auth_option_params(user, params)
```

Comment:
Should these new functions be in a service instead of a policy because they're mutating user objects? Not sure about this as I'm still learning the policy/service/query pattern.